src/main/java/com/salesforce/storm/spout/dynamic/persistence/zookeeper/CuratorFactory.java

```diff
@@ -88,7 +88,7 @@ public static CuratorFramework createNewCuratorInstance(final Map<String, Object
             // Create new ThreadFactory with named threads.
             // TODO allow pushing in better naming.
             final ThreadFactory threadFactory = new ThreadFactoryBuilder()
-                .setNameFormat("[DynamicSpout:PersistenceAdapter] Curator Pool %d")
+                .setNameFormat("[DynamicSpout:" + context + "] Curator Pool %d")
```

Comment:
Instead of DynamicSpout why not Curator?

Reply:
See reasoning above about using a common prefix